config/foreman.hiera/common.yaml

```diff
@@ -19,4 +19,4 @@ foreman::config::apache::proxy_params:
   retry: '0'
   timeout: '900'
 
-katello::globals::enable_katello_agent: "%{alias('foreman_proxy_content::enable_katello_agent')}"
+katello::globals::enable_katello_agent: false
```

Comment:
In https://github.com/theforeman/puppet-katello/pull/471 this parameter is removed, so it can be dropped.

Reply:
Thanks!